server/src/main/java/io/seata/server/session/SessionHolder.java

```diff
@@ -380,14 +379,7 @@ public static boolean distributedLockAndExecute(String key, NoArgsFunc func) {
     }
 
     public static void destroy() {
-        Collection<RaftServer> raftServers = RaftServerFactory.getInstance().getRaftServers();
-        if (raftServers != null) {
-            try {
-                raftServers.forEach(RaftServer::close);
-            } catch (Exception e) {
-                LOGGER.error(e.getMessage());
-            }
-        }
+        RaftServerFactory.getInstance().destroy();
```

Comment:
In the future, it is necessary to refactor the code to decouple SessionHolder with RaftXXX.

Reply:
> In the future, it is necessary to refactor the code to decouple SessionHolder with RaftXXX.

Since the raft cluster must match the raft mode exactly, it must be written in the sessionHolder, otherwise the timing of the initialization is not very controllable, and the raft cluster must be started after the sessionmanager has finished initializing, otherwise there may be npe issues